rules/file-contents-config.json

```diff
@@ -17,6 +17,11 @@
     "fail-on-non-existent": {
       "type": "boolean",
       "default": false
+    },
+    "display-line-numbers": {"type": "boolean", "default": false},
+    "context-length": {
```

Comment:
I really want it to be clear that this is a unit of characters, but I'm unsure on how. There's the documentation side, but would be nice if folk could read a rule and know that this is chars and not bytes or words or lines.

Thinking chars... how about context-width? That at least gives a sideways feeling and not vertical feeling.

Reply:
Are context and display-line-numbers paired and both required when used? i.e. wondering what to do in the requirement logic below.

Reply:
Updated PR.
I think `display-result-context` fits well here.